Fix alignment news section about page 1498

celinevalentine opened the pull request:
Fixes #1498 

### What changes did you make and why did you make them ?

  -added a class for news images to make it easier to resize and have the same height
  -for the news-cells class, commented out column nowrap property as it didn't do anything 
  -for the more-padding class, added a padding-bottom to align paragraphs better

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->
<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->
<img width="891" alt="Screen Shot 2021-07-25 at 12 17 05 AM" src="https://user-images.githubusercontent.com/52294389/127034183-05da585a-461c-4bf0-a481-fa84bba808eb.png">


<details>
<summary>Visuals before changes are applied</summary>

![image](Paste_Your_Image_Link_Here_After_Attaching_Files)
<img width="793" alt="Screen Shot 2021-07-26 at 11 10 34 PM" src="https://user-images.githubusercontent.com/52294389/127104645-368bf0b9-e51e-4d64-9443-9d80778ca6e2.png">


</details>
I had trouble lining up paragraphs so all three are at the same level horizontally (top baseline). I would handle it better if the images and paragraphs are not dynamic. I'm continuing searching for the better solution to align paragraphs perfectly. But as for now, I'd like to submit my draft for comments and feedback.
<details>

celine-valentine committed:
updated employee icon title link

celine-valentine committed:
added news-images and fixed news-cells and more-padding alignments

the paragraphs are still not perfectly aligned horizontally

celinevalentine commented:
Also, do I need to delete this commit as below? I first pushed it to the wrong branch but revert it. I'm not sure if I did correctly.

updated employee icon title link
260e115

adam-abundis commented on _data/internal/credits/employees.yml:
```diff
@@ -1,6 +1,6 @@
 ---
 title: Employees
-title-link: https://thenounproject.com/
+title-link: https://thenounproject.com/search/?creator=1145943&q=employees&i=3201878
```
This seems to be from a different issue. Please remove the changes from this pull request and push the changes.

celinevalentine replied:
Got it, thanks! Just pushed a new commit. 

How do I remote that commit? Can I do git reset --hard?

celinevalentine replied:
I thought I reverted that commit last week by following the link you sent me but I guess it's still there for some reason.

adam-abundis reviewed (changes requested):
Hi @celinevalentine. Good job resolving your branch issue so you could make a pull request. Yes, please remove 260e115 since this does not resolve the issue. Also, it seems like you made a class `news-images` for styling purposes, but you did not add the class to [line 7 of website/pages/about/about-card-news.html](https://github.com/hackforla/website/blob/a82d9759b218a0bacbf6624c57e8658da4ccbcda/pages/about/about-card-news.html#L7). Add the news-images` class here, and the styling for the images will take effect. 

If you have any questions, feel free to comment here or feel free to reach me on Slack.

celine-valentine committed:
added news-images class in the html file

also deleted the changes on employee.yml file from a different commit

macho-catt reviewed (changes requested):
Hello Celine. Good job working on this!  It's looking good so far.

However, there are some other changes that I would like to request. Specifically, there are still some work to do to match the figma design (attaching @abuna1985 's photo of the figma), and the end goal is to have it look as close as possible to this:

![127014524-fe354f21-c095-4984-baa5-b81e6c88d879](https://user-images.githubusercontent.com/48004150/129309297-ba444be4-4039-4ec9-9ac8-589dee94ab7d.jpg)

Right now in your request, the 'containers' for the three cells have different dimensions. We want to make sure the height and widths are the same, so the three cells are uniform.

<details>
<summary>Click for photos</summary>

![Screenshot 2021-08-12 22:20:39](https://user-images.githubusercontent.com/48004150/129309424-ae10e335-bc82-4fcd-8f42-99d4be31b1bb.png)

![Screenshot 2021-08-12 22:24:31](https://user-images.githubusercontent.com/48004150/129309606-200d58d5-6975-463c-abe6-5078f6878803.png)

![Screenshot 2021-08-12 22:24:41](https://user-images.githubusercontent.com/48004150/129309627-97aecced-b62c-4930-95af-9100d6658be2.png)

![Screenshot 2021-08-12 22:24:50](https://user-images.githubusercontent.com/48004150/129309637-bc4979cf-4f14-4aaa-943e-8d11c470b731.png)

</details>

Additionally, the paragraphs should be aligned. Right now, they do not.

<details>
<summary>Text lines are not aligned</summary>

![line height](https://user-images.githubusercontent.com/48004150/129310886-d30210b2-988d-4d3d-b860-de8afaf90f17.png)


</details>

Lastly, tablet and mobile views are not responsive atm:

<details>

<summary>Tablet</summary>

![tablet](https://user-images.githubusercontent.com/48004150/129313011-ec82d2b6-3909-4e9a-829f-cea8c2546ed2.png)


</details>

<details>

<summary>Mobile</summary>

![mobile](https://user-images.githubusercontent.com/48004150/129313033-30a88691-1219-48d2-a5bc-0c619ee7b4c5.png)



</details>

So my suggestions would be:
1) Make sure news-cells have a set height and width so that it is uniform for all three. In my sample, I set the height and width to something like 250px by 250px. Play around and see what works.

<details>
<summary>Sample</summary>

![sample](https://user-images.githubusercontent.com/48004150/129311893-14aaec66-9f7d-4565-933e-176e3bbfa13e.png)


</details>

2. This also kind of fixes the alignment issue of the paragraph text. They are better aligned because the container has a set height and width. The LA times one, which wraps into three lines, still does not start on the same line, but at least it is on the same line as the two other paragraph texts. There are other adjustments that can be done here to achieve the same look as the figma
3. The image ratios would need some fixing since the CNN looks bigger and LA times is smaller
4. Mobile responsiveness for tablet and mobile looks like they are using flex and centered. However, I'm not sure if the issue was supposed to cover responsiveness. @abuna1985 , what do you think? Should this be a separate issue instead?

You're almost there!

macho-catt commented on _sass/components/_about.scss:
```diff
@@ -580,21 +580,26 @@ a.anchor {
 .news-cells {
   display: flex;
-  flex-flow: column nowrap;
+  // flex-flow: column nowrap;
```
since this is a commented line, you can remove this if it's unused

celinevalentine replied:
Hi Matt, thanks so much for your comments and feedback. I'll revise it and submit it again. I'll use mobile first approach for the responsiveness. I think my current struggle is to align the text due to the nature of dealing with the dynamic output. My reasoning was to align the images to the bottom baseline so that texts below them can start at the same level. But so far, it's not working the way I wanted. I'll keep trying.

adam-abundis commented:
@celinevalentine Thank you for your patience. I know you have been working on this issue for a while now. A huge shoutout/thank you to @macho-catt for providing productive feedback.

In the case of responsiveness, #1498 seems to request the news section match the Figma design (desktop/mobile). @celinevalentine I can create an issue for tablet view if you would like. If not, then make the edits so it is fully responsive. Let me know on here or on the HfLA slack. I am also here if you need to talk out a solution for @macho-catt recommendations.

celine-valentine committed:
aligned .news-cells p with responsiveness

celinevalentine commented:
<img width="813" alt="Screen Shot 2021-08-20 at 11 07 39 PM" src="https://user-images.githubusercontent.com/52294389/130312564-fabaac12-7d83-42b7-b4c4-cef547ff6267.png">
Just pushed the 4th commit. To solve the alignment issue of the paragraphs, I fixed the width and height of the images above them then changed .news-cells align-items to flex-start. I changed it back to center for the mobile and tablet views. Let me know if the images are still not the size like in Figma.
<img width="771" alt="Screen Shot 2021-08-20 at 11 07 18 PM" src="https://user-images.githubusercontent.com/52294389/130312545-593a3d6e-6a36-462e-ae79-970b66f77448.png">

Aveline-art commented on _sass/components/_about.scss:
```diff
@@ -580,12 +580,26 @@ a.anchor {
+  
+}
+
+// .more-padding {
+//   padding-top: 40px;
+//   margin: 0 20px 0 20px;
+//   padding-bottom: 10px;
+// }
+
```
Does this need to be here?

celinevalentine replied:
No, I removed it and pushed to my branch.

Aveline-art reviewed (commented):
Hi @celinevalentine. Thank you for bearing with us as we review this. Everything looks good to me. If you can resolve the merge conflicts, and clean up the commented out code, I will merge this asap. Thank you for sticking with this.

macho-catt commented:
@macho-catt will reach out to @celinevalentine and possibly do a pair programming

celine-valentine committed:
removed unused commented out code

celinevalentine commented:
Could anyone tell me how to find out which line of codes have the conflicts?

macho-catt commented:
@celinevalentine I messaged you on slack about this, but generally you can find merge conflicts at the bottom of the page where it says "Conflicting files." In your case:

Conflicting files
_sass/components/_about.scss
pages/about/about-card-news.html

DM me on slack if you need further help.

macho-catt commented:
Hi @celinevalentine. Reaching out again to see if you can fix the merge conflicts. Please DM me if you need help. Otherwise, I will close this PR in 7 days since it's been in limbo for a couple of months. Thanks.

macho-catt commented:
Celine informed me that she won't be able to finish the issue. Closing the PR.

BrianCodes33 commented:
One quick solution I thought of:
give second image a negative margin
`margin-top: -2em`
also the second p element of 
`margin-left 2 rem`
lastly change the last anchor element to 
`position: relative;
top: 20px;`
i also think the anchor elements (links) look better centered. here is the screenshot while working in my dev tools. let me know if this is an acceptable solution.
<img width="875" alt="Screen Shot 2022-01-20 at 9 52 09 AM" src="https://user-images.githubusercontent.com/13526442/150394347-4801e28e-1031-43d9-99f5-e8d86c6f1654.png">
